interface-definitions/interfaces_wireless.xml.in

```diff
@@ -428,6 +419,139 @@
+                      <leafNode name="freq-2">
+                        <properties>
+                          <help>HE operating channel center frequency - center freq 2 (for use with the 80+80 mode)</help>
+                          <valueHelp>
+                            <format>u32:1-233</format>
+                            <description>6Ghz (802.11 ax) center channel index (use 23 (at 80MHz) for secondary channel 17)</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-233"/>
+                          </constraint>
+                          <constraintErrorMessage>Channel center value must be between 1 and 233</constraintErrorMessage>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <leafNode name="antenna-pattern-fixed">
+                    <properties>
+                      <help>Set if antenna pattern does not change during the lifetime of an association</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <node name="beamform">
+                    <properties>
+                      <help>HE Beamforming capabilities</help>
+                    </properties>
+                    <children>
+                      <leafNode name="single-user-beamformer">
+                        <properties>
+                          <help>Support for operation as single user beamformer</help>
+                          <valueless/>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="single-user-beamformee">
+                        <properties>
+                          <help>Support for operation as single user beamformee</help>
+                          <valueless/>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="multi-user-beamformer">
+                        <properties>
+                          <help>Support for operation as multi user beamformer</help>
+                          <valueless/>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <leafNode name="bss-color">
+                    <properties>
+                      <help>BSS color (1-63)</help>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-63"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                </children>
+              </node>
+              <leafNode name="require-he">
+                <properties>
+                  <help>Require stations to support HE PHY (reject association if they do not)</help>
```

Comment:
Similar to the VHT comment above — let's get rid of the parenthesized part.

Reply:
I have addressed your remarks in commit [74736d4](https://github.com/vyos/vyos-1x/pull/3524/commits/74736d4ca0e7e00799abe03d3f78a43eeee457e9).